src/pages/Information/Information.jsx

```diff
@@ -1,57 +1,55 @@
 import React from "react";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
 import { FormattedMessage } from "react-intl";
+import InformationData from "./InformationData";
 
-const Information = () => (
-  <>
-    <div data-test="information-display">
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-first-header"
-          defaultMessage="What is HacktoberFest?"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-first-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-      <img
-        data-test="image-display"
-        src="../src/assets/pumpkin_PNG86719.png"
-        alt="hacktoberfest"
-        style={{ height: 100 }}
-      />
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-second-header"
-          defaultMessage="When & Where"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-second-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-third-header"
-          defaultMessage="Come join the FUN!"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-third-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-  </>
-);
+const Information = () => {
+  const displayInfo = () => {
```

Comment:
:question: It looks like you've created some sort of higher order component. Is that what we really want here?

Reply:
Addressed:  ee24bb572ba3b2cd2af4237956ad9b62932577f5